browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/extensions/api/settings_private/brave_prefs_util.h"
+
+#include "brave/common/pref_names.h"
+#include "chrome/browser/extensions/api/settings_private/prefs_util.h"
+#include "chrome/common/pref_names.h"
+#include "chrome/common/extensions/api/settings_private.h"
+
+namespace extensions{
+
+namespace settings_api = api::settings_private;
+
+const PrefsUtil::TypedPrefMap& BravePrefsUtil::GetWhitelistedKeys() {
+  // Static cache, similar to parent class
+  static PrefsUtil::TypedPrefMap* s_brave_whitelist = nullptr;
```

Comment:
Do we want `s_brave_whitelist` in a `std::unique_ptr` or something like that? Or does it not matter for this?

Reply:
Discussed in PM - followed parent class pattern of using static variable that hangs around for the whole browser process life, seems ok since this is a global whitelist.